tests/UpgradeRule/PHP/AddNamespaceTest.php

```diff
@@ -146,4 +146,42 @@ public function testPsr4Option()
         $this->assertFalse($changeset->hasWarnings($file4->getPath()));
         $this->assertEquals($output4, $generated4);
     }
+
+    /**
+     * Test applying namespaces to a file using scalar parameter type and return types.
+     */
+    public function testNamespaceUseStatement()
+    {
+        list($parameters, $input1, $output1) =
```

Comment:
Some of these variables do not need to be numbered.

Reply:
Fixed.